js_modules/dagster-ui/packages/ui-core/src/nav/LeftNavItemType.tsx

```diff
@@ -9,7 +9,7 @@ import {RepoAddress} from '../workspace/types';
 export type LeftNavItemType = {
   name: string;
   isJob: boolean;
-  leftIcon: IconName;
+  leftIcon: IconName | 'airflow';
```

Comment:
I think unnecesary

Reply:
Urgh this stuff is from an attempted impl. The issue with it is that it strips color from the svg. So I had to switch to using the TagIcon.